Lib/fontParts/base/groups.py

```diff
@@ -160,186 +236,269 @@ def _get_side2KerningGroups(self):
+
+        Example::
 
             >>> font.groups["myGroup"] = ["A", "B", "C"]
+
         """
+
         super(BaseGroups, self).__setitem__(groupName, glyphNames)
 
-    def clear(self):
-        """
-        Removes all group information from Groups,
-        resetting the Groups to an empty dictionary. ::
+    def clear(self) -> None:
+        """Remove all groups from the current groups instance.
+
+        This will reset the :class:`BaseGroups` instance to an empty dictionary.
+
+        Example::
 
             >>> font.groups.clear()
+
         """
         super(BaseGroups, self).clear()
 
-    def get(self, groupName, default=None):
-        """
-        Returns the contents of the named group.
-        **groupName** is a :ref:`type-string`, and the returned values will
-        either be :ref:`type-immutable-list` of group contents or ``None``
-        if no group was found. ::
+    def get(
+        self, groupName: str, default: Optional[CollectionType[str]] = None
+    ) -> Optional[Tuple[str, ...]]:
+        """Get the contents for the given group in the current groups instance.
+
+        If the given `groupName` is not found, The specified `default` will be
+        returned.
+
+        :param groupName: The group name to look up as a :class:`str`.
+        :param default: The optional default value to return if the `groupName`
+            is not found`. The value must be either a class`list` or :class:`tuple`
+            of :class:`str` glyph names, or :obj:`None`. Defaults to :obj:`None`.
+        :return: The contents of the given group as a :class:`tuple`
+            of :class:`str` items, or the `default` value if the group is not found.
+
+        Example::
 
             >>> font.groups["myGroup"]
             ("A", "B", "C")
 
-        It is important to understand that any changes to the returned group
-        contents will not be reflected in the Groups object. If one wants to
-        make a change to the group contents, one should do the following::
+        ..note::
+
+            Any changes to the returned lib contents will not be reflected in
+            it's :class:`BaseGroups` instance. To make changes to this content,
+            do the following::
+
+                >>> group = font.groups["myGroup"]
+                >>> group.remove("A")
+                >>> font.groups["myGroup"] = group
 
-            >>> group = font.groups["myGroup"]
-            >>> group.remove("A")
-            >>> font.groups["myGroup"] = group
         """
         return super(BaseGroups, self).get(groupName, default)
 
-    def items(self):
-        """
-        Returns a list of ``tuple`` of each group name and group members.
-        Group names are :ref:`type-string` and group members are a
-        :ref:`type-immutable-list` of :ref:`type-string`. The initial
-        list will be unordered.
+    def items(self) -> BaseItems[Tuple[str, ValueType]]:
+        """View the items in the current groups instance.
```

Comment:
As per other PR, maybe Return instead of View

Reply:
Fixed.